web/styles/dark_theme.css

```diff
@@ -378,6 +378,10 @@
         }
     }
 
+    .stream_header_colorblock.disabled {
+        filter: brightness(50%);
+    }
```

Comment:
Why is this in the dark theme only?

Reply:
The colorblock is darkened in dark theme to match the color of the stream symbol. In the light theme, it matches the color of the stream symbol. So this styling is not needed.

Here is the color when I use this in light theme as well:
![Screenshot 2024-02-14 103908](https://github.com/zulip/zulip/assets/119798962/bd906cdf-b669-440a-8fcb-610bc9e5aad9)

Reply:
OK, that makes sense.